packages/sdks/src/functions/get-fetch.ts

```diff
@@ -1,12 +1,21 @@
 import { getGlobalThis } from './get-global-this.js';
 
+/**
+ * The issue is that when building standalone bundle the bundler includes `node-fetch` as
```

Comment:
which bundler are we talking about here? Is this a qwik optimizer/bundler issue in particular? or a Vite issue? can we add that information for clarity?

Reply:
clarified